### PR TITLE
Added search support for android market (playstore)

### PR DESCRIPTION
**Pull request by sesubash**
Added search support for android market (playstore)

### CODE REVIEW

**Comment by zaidakram** [comment]
Need this merged. Its a solid pull request. Working for me. Thanks

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Currently this plugin is compatible only with Android and iOS. Feel free to add
 
 This plugin follows the Cordova 3.0 plugin spec, so it can be installed through the Cordova CLI in your existing Cordova project:
 
-    cordova plugin add https://github.com/xmartlabs/cordova-plugin-market
+    cordova plugin add https://github.com/sesubash/cordova-plugin-market
```
Comment (enrigalmig) [inline README.md]: Please do not change this :)

**Comment by enrigalmig** [comment]
This is looking good, please squash your commits into 1. Waiting for @m-revetria review

File: README.md (hunk 1 above)
Comment (sesubash) [inline README.md]: Updated read me :)

**Comment by m-revetria** [comment]
@zaidakram: Everything is looking good, please squash your commits before we merge this PR. 

Thanks for your contribution!

**Comment by enrigalmig** [comment]
Hey @sesubash, please follow this guide https://github.com/ginatrapani/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit.

Thank you!

**Comment by anonrig** [comment]
Why don't we get this merged using the new github features?

**Comment by m-revetria** [comment]
@anonrig good point! 
Thanks!

**Pull request merged**